atlasdb-dbkvs/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/OracleTableNameMapperTest.java

```diff
@@ -87,6 +89,20 @@ public void shouldThrowIfTable99999Exists() {
         oracleTableNameMapper.getShortPrefixedTableName(connectionSupplier, TEST_PREFIX, tableRef);
     }
 
+    @Test
+    public void shouldGetRightPrimaryKeyConstraintForLongTableNames() {
+        String pkConstraintName = OraclePrimaryKeyConstraintNames.get(LONG_TABLE_NAME);
+        assertThat(pkConstraintName, is(StringUtils.left("pk_" + LONG_TABLE_NAME, 30)));
+        assertThat(pkConstraintName.length(), is(OracleTableNameMapper.ORACLE_MAX_TABLE_NAME_LENGTH));
```

Comment:
They will end up having the same constraint names in the current impl. We will have fix for this in develop. Filed #1551 and will fix it next